app/jobs/publish_feed_job.rb

```diff
@@ -12,7 +12,9 @@ def perform(podcast)
     setup_data(podcast)
     @rss = generate_rss_xml
     save_podcast_file(@rss)
-    copy_podcast_file_alias if podcast.feed_rss_alias.present?
+    if podcast.default_feed.file_name != Feed::DEFAULT_FILE_NAME
+      copy_podcast_file_alias
```

Comment:
I nuked the `podcasts.feed_rss_alias` column, but copied it into the default feed's `file_name`.  These are used for feed CDNs that need/want a plain path to the RSS, not a `.xml` extension.

For temporary backwards compatibility - every podcast still writes to `feed-rss.xml`, but these few with aliases also copy to them.  I need to check if any shows need _both_ those files - but may be safe to just write to the one.